### ISSUE TITLE
Stars in Dark background gets something like light border

### ISSUE THREAD

**Issue by jd-0001** [craigh411/vue-star-rating#56]
Hello,

I have used your star component it is awesome. However, I am getting UI issue when dark background is used.

Please check this image:
![dark-bg](https://user-images.githubusercontent.com/47495003/68667073-26deda00-056b-11ea-8ace-33c5ed95cdbb.png)

I also get white stroke in yellow stars, if size is small. Try changing code for glowing stars in your jsfiddle example to below code.
```
  <h2>Glowing Stars</h2>
  <div style="background:#000;padding-bottom:10px;">

    <star-rating :star-size="20"></star-rating>
  </div>
```
Result shade of light grey or white:
![glow](https://user-images.githubusercontent.com/47495003/68667216-71f8ed00-056b-11ea-967d-a3b233466ad0.png)

I am also getting same in my project which is live. You can check how it looks in below image.
![dist](https://user-images.githubusercontent.com/47495003/68667292-a40a4f00-056b-11ea-9ca4-d7acce28c42e.png)

However, this might looks tiny issue but if we use dark theme in our app as eveybody loves now a days it brings this issue.

Awesome Component BTW.
Waiting for your response.

~ Regards

**Comment by ivansteff** [comment]
+ 
I have same problem

**Comment by Aurion72** [comment]
I have it too

**Comment by cj4luv** [comment]
https://github.com/craigh411/vue-star-rating/blob/6122c1510e2727423551d2dd5863628cdec4fe77/src/star.vue#L17

this line stroke prop name glowColor

**Comment by craigh411** [comment]
Fixed in version 1.6.3
